pep-0698.rst

```diff
@@ -472,3 +471,7 @@ Copyright
 
 This document is placed in the public domain or under the
 CC0-1.0-Universal license, whichever is more permissive.
+
+.. |overrides-library| replace:: ``overrides`` library
+
+.. _overrides-library: https://pypi.org/project/overrides/
```

Comment:
```suggestion
```

As discussed above, all this extra complexity really doesn't seem necessary or prudent.